tests/testdata/ilimodels/PipeBasketTest_V1.ili

```diff
@@ -5,6 +5,7 @@ AT "https://signedav.github.io/usabilitydave/models"
 VERSION "2020-06-22" =
 
   STRUCTURE Address =
+    JustARandomEnum: (One, Two, Three);
```

Comment:
There is already a test checking how many topics are found in this model (2). With this ENUM it would have found 3 what would have given an error. But with this implementation it still finds 2. So the tests can stay the same.